docs/source/models.rst

```diff
@@ -1122,6 +1122,194 @@ in native spaCy format.
 
 For more information, see :py:mod:`mlflow.spacy`.
 
+``Spacy`` pyfunc usage
+~~~~~~~~~~~~~~~~~~~~~~
+
+The example below shows how to train a ``Spacy`` ``TextCategorizer`` model, log the model artifact and metrics to the
+mlflow tracking server and then load the saved model to make predictions. For this example, we will be using the
+``Polarity 2.0`` dataset available in the ``nltk`` package. This dataset consists of 10000 positive and 10000 negative
```

Comment:
Does this example finish quickly? Curious how long it takes to preprocess data and train a model.

Reply:
It takes about 10 minutes to train this particular with the default number of training steps in the spacy config file (2000).

Reply:
10 minutes is long. Can we reduce the training data size or the number of training steps?

Reply:
Sure, added the instruction to reduce the number of training steps to 100 (default is 20000, not 2000) and the eval frequency to 10 (default is 100). Now it completes in 1-2 minutes.

Reply:
https://github.com/mlflow/mlflow/pull/7753/files#diff-27092883ef9a4a81ab9b5ea59442ab48dd314babd61428270948fcfae0a80f28R1273-R1277

Reply:
> Now it completes in 1-2 minutes.

Great!